src/cli/keepassxc-cli.1

```diff
@@ -147,4 +147,5 @@ Use extended ASCII characters for the generated password. [Default: Disabled]
 Bugs and feature requests can be reported on GitHub at https://github.com/keepassxreboot/keepassxc/issues.
 
 .SH AUTHOR
-This manual page was written by Manolis Agkopian <m.agkopian@gmail.com>.
+This manual page was originally written by Manolis Agkopian <m.agkopian@gmail.com>,
+as is maintained by the KeePassXC Team <team@keepassxc.org>.
```

Comment:
"as is"
Should this be "and is" instead?

Reply:
nice catch! fixed.